CONTRIBUTING.rst

```diff
@@ -57,12 +57,41 @@ If you are proposing a feature:
 Get Started!
 ------------
 
-Ready to contribute? Here's how to set up `enterprise` for local development.
+Ready to contribute? Here's how to set up ``enterprise`` for local development.
 
-1. Fork the `enterprise` repo on GitHub.
+Install the dependencies
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+``enterprise`` relies on a lot of other software to function.
+If you use the Anaconda distribution of Python, you can get all of this software using ``conda``.
+First, you install the latest stable version of ``enterprise``, which will come with all of the dependencies.
+Then you remove ``enterprise`` leaving everything else intact.
+This way you can use your development version of ``enterprise`` instead of the stable version.
+We will also need some additional software that is required to run the tests.
+
+Start with an empty virtual environment, in this case called ``ent_dev``::
+
+    $ conda create -n ent_dev -y -c conda-forge python=3.9
+    $ conda activate ent_dev
+
+Now install things by running the commands::
+
+    $ conda install -c conda-forge enterprise-pulsar
+    $ conda remove enterprise-pulsar --force
+    $ conda install -c conda-forge  black=22.3.0 flake8 sphinx_rtd_theme pytest-cov
```

Comment:
Is there any reason not to put all the conda requirements on the `conda create` line? That should save some time.